active/AnimatedIcon/AnimatedIconSpec.md

```diff
@@ -0,0 +1,363 @@
+                                        <VisualStatex:Name="Normal">
+                                            <VisualState.Setters>
+                                                <Setter Target="Icon.(AnimatedIcon.StateProperty)"Value="Normal"/>
+                                            </VisualSTate.Setters>
+                                        </VisualState>
+                                        <VisualState x:Name="PointerOver">
+                                            <VisualState.Setters>
+                                                <Setter Target="Icon.(AnimatedIcon.StateProperty)"Value="PointerOver"/>
+                                            </VisualSTate.Setters>
+                                        </VisualState>
+                                        <VisualStatex:Name="Pressed">
+                                            <VisualState.Setters>
+                                                <Setter Target="Icon.(AnimatedIcon.StateProperty)"Value="Pressed"/>
+                                            </VisualSTate.Setters>
+                                        </VisualState>
+                                        <VisualStatex:Name="Disabled">
+                                            <VisualState.Setters>
+                                                <Setter Target="Icon(AnimatedIcon.StateProperty)"Value="Disabled"/>
+                                            </VisualSTate.Setters>
+                                        </VisualState>
+                                    </VisualStateGroup>
+                                </VisualStateManager.VisualStateGroups>
+                                <ContentPresenter
+                                    x:Name="ContentPresenter"
+                                    Background="{​​​​​​​​​TemplateBinding Background}​​​​​​​​​"
+                                    BorderBrush="{​​​​​​​​​TemplateBinding BorderBrush}​​​​​​​​​"
+                                    BorderThickness="{​​​​​​​​​TemplateBinding BorderThickness}​​​​​​​​​"
+                                    Content="{​​​​​​​​​TemplateBinding Content}​​​​​​​​​"
+                                    ContentTemplate="{​​​​​​​​​TemplateBinding ContentTemplate}​​​​​​​​​"
+                                    ContentTransitions="{​​​​​​​​​TemplateBinding ContentTransitions}​​​​​​​​​"
+                                    Padding="{​​​​​​​​​TemplateBinding Padding}​​​​​​​​​"
+                                    HorizontalContentAlignment="{​​​​​​​​​TemplateBinding HorizontalContentAlignment}​​​​​​​​​"
+                                    VerticalContentAlignment="{​​​​​​​​​TemplateBinding VerticalContentAlignment}​​​​​​​​​"
+                                    AutomationProperties.AccessibilityView="Raw" />
+                                <Grid x:Name="Icon" HorizontalAlignment="Right" Content="{​​​​​​​​​TemplateBinding Icon}​​​​​​​​​/>
```

Comment:
Grid doesn't have a Content property. Is this supposed to be a ContentControl or ContentPresenter?